shoop/simple_pricing/migrations/0002_remove_simpleproductprice_includes_tax.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('simple_pricing', '0001_initial'),
```

Comment:
Good idea, but if no such data migration is created, this does not matter. The SimpleProductPrice.includes_tax was not visible in the UI so I don't think many have set it manually. Therefore we could as well use the same default value for Shop.prices_include_tax and even if that turns out to be wrong for some case, it's usually not a big deal to manually fix couple booleans from the Admin (you're not having millions of Shops in a single Shoop instance -- hopefully).